### PR TITLE
gst: Render tracer objects

### PR DESCRIPTION
**Pull request by thiblahute**
Useful once https://gitlab.freedesktop.org/gstreamer/gstreamer/-/merge_requests/8086 is merged

### CODE REVIEW

**Comment by tp-m** [comment]
Will this make tracer action signals show up in the docs as well by any chance?

**Comment by thiblahute** [comment]
> Will this make tracer action signals show up in the docs as well by any chance?

Yes, it does.

**Comment by MathieuDuponchelle** [comment]
thanks, will review next week :)

File: hotdoc/extensions/gst/gst_extension.py [hunk 1]
```diff
@@ -864,7 +864,7 @@ def __create_property_symbols(self, obj, parent_uniquename,
             type_ = QualifiedSymbol(type_tokens=tokens)
 
             default = prop.get('default')
-            if obj['hierarchy'][0] != parent_uniquename:
```
Comment (MathieuDuponchelle) [inline hotdoc/extensions/gst/gst_extension.py:867]: I'm missing some context here, can you elaborate on why this is needed / give an example in the commit message?

**Comment by MathieuDuponchelle** [comment]
lgtm otherwise, please merge once comment is addressed :)

**Pull request merged**